<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InDSSVLopTinChi</class>
 <widget class="QDialog" name="InDSSVLopTinChi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1019</width>
    <height>645</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="Quaylai">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>90</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Quay lại</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>40</y>
     <width>491</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Danh sách sinh viên đã đăng ký lớp tín chỉ</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableDSSV">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>150</y>
     <width>971</width>
     <height>471</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
   </property>
   <property name="columnCount">
    <number>7</number>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>125</number>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Mã lớp tín chỉ</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Mã SV</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Mã môn</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Tên môn</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Niên khóa</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Học kỳ</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nhóm</string>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="EditNienKhoa">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="EditHocKy">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>110</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="EditNhom">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>110</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="EditMaMon">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>110</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelHocKy">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>110</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Học kỳ</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelNhom">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>110</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Nhóm</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelMaMon">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>110</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Mã môn</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelNienKhoa">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Niên khóa</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
